Doc/library/copy.rst

```diff
@@ -89,6 +97,20 @@ with the component as first argument and the memo dictionary as second argument.
 The memo dictionary should be treated as an opaque object.
 
 
+.. index::
+   single: __replace__() (replace protocol)
+
+Function :func:`replace` is more limited than :func:`copy` and :func:`deepcopy`,
+and only supports named tuples, dataclasses, and other classes which
```

Comment:
```suggestion
and only supports :term:`named tuples <named tuple>`, :mod:`dataclasses`, and other classes which
```

Reply:
Actually, it does not support named tuples in general. It currently only supports named tuples created by `collections.namedtuple()`.